packages/device_info_plus/device_info_plus/example/integration_test/device_info_plus_test.dart

```diff
@@ -65,4 +66,121 @@ void main() {
       }
     }
   });
+
+  testWidgets('iOS', (WidgetTester tester) async {
+    if (Platform.isIOS) {
+      expect(iosInfo.name, isNotNull);
+      expect(iosInfo.systemName, equals('iOS'));
+      expect(iosInfo.systemVersion, isNotNull);
+      expect(iosInfo.model, equals('iPhone'));
+      expect(iosInfo.localizedModel, equals('iPhone'));
+
+      expect(iosInfo.identifierForVendor, isNotNull);
+      expect(iosInfo.isPhysicalDevice, isNotNull);
+
+      expect(iosInfo.utsname.sysname, equals('Darwin'));
+      expect(iosInfo.utsname.nodename, isNotNull);
+      expect(iosInfo.utsname.release, isNotNull);
+      expect(iosInfo.utsname.version, isNotNull);
+      expect(iosInfo.utsname.machine, isNotNull);
+    }
+  });
+
+  testWidgets('android', (WidgetTester tester) async {
+    if (Platform.isAndroid) {
+      expect(androidInfo.version.baseOS, isNotNull);
+      expect(androidInfo.version.codename, isNotNull);
+      expect(androidInfo.version.incremental, isNotNull);
+      expect(androidInfo.version.previewSdkInt, isNotNull);
+      expect(androidInfo.version.release, isNotNull);
+      expect(isInteger(androidInfo.version.sdkInt), true);
+      expect(androidInfo.version.securityPatch, isNotNull);
```

Comment:
Instead of checking for `notNull`, which is not bad, but could be better, can you check against actual values?

for example: `expect(androidInfo.version.sdkInt, 30)`

We can do that because we know that the Emulator has version 30